src/lua/protobuf_wireformat.lua

```diff
@@ -106,6 +173,21 @@ local function encode_tag(field_id, wire_type)
     return encode_varint(bit.bor(bit.lshift(field_id, 3), wire_type))
 end
 
+-- Decode a tag byte.
+local function decode_tag(ctx)
+    local tag = decode_int(ctx)
+    return bit.band(tag, 0x7), bit.rshift(tag, 3)
+end
+
+-- Return default value for input type.
+local function set_default(type)
```

Comment:
This function don't set a default to anywhere, it just returns it, so the name seems confusing. It could be `get_default`, I guess.